src/server/qgsserversettings.cpp

```diff
@@ -615,3 +638,23 @@ QString QgsServerSettings::serviceUrl( const QString &service ) const
 
   return result;
 }
+
+int QgsServerSettings::projectCacheCheckInterval() const
+{
+  return value( QgsServerSettingsEnv::QGIS_SERVER_PROJECT_CACHE_CHECK_INTERVAL ).toInt();
+}
+
+QString QgsServerSettings::projectCacheStrategy() const
+{
+  QString result = value( QgsServerSettingsEnv::QGIS_SERVER_PROJECT_CACHE_STRATEGY ).toString();
+  if ( !result.compare( QLatin1String( "filesystem" ) ) ||
+       !result.compare( QLatin1String( "periodic" ) ) ||
+       !result.compare( QLatin1String( "off" ) ) )
+  {
+    QgsMessageLog::logMessage( QStringLiteral( "Invalid cache strategy, expecting 'filesystem', 'periodic' or 'off'. Using 'filesystem' as default." ), "Server", Qgis::MessageLevel::Warning );
+    result = QString( "filesystem" );
```

Comment:
@m-kuhn What is preferable in this case: `QStringLiteral` or `QLatin1String`? I always forget :(.

Reply:
```suggestion
    result = QStringLiteral( "filesystem" );
```

Quoting [Qt](https://www.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral)
> Use QStringLiteral only if a QString needs to be created

I think we are clearly creating a `QString` here

Reply:
@dmarteau this suggestion is good to apply

Reply:
Fixed